client/app/src/androidTest/java/com/example/criengine/TestUtilityMethods.java

```diff
@@ -2,16 +2,66 @@
 
 import android.widget.EditText;
 
+import com.example.criengine.Activities.LoginActivity;
+import com.example.criengine.Activities.RootActivity;
 import com.robotium.solo.Solo;
 
+import static junit.framework.TestCase.assertTrue;
+
 /**
  * Contains utility methods used by the tests.
  */
 public class TestUtilityMethods {
+    /**
+     * Log in with the first intentTesting Account.
+     * @param solo The solo object.
+     */
+    public static void login1(Solo solo) {
+        // Asserts that the current activity is the LoginActivity.
+        solo.assertCurrentActivity("Wrong Activity", LoginActivity.class);
+
+        // Input username and password
+        solo.enterText((EditText) solo.getView(R.id.loginEditTextEmail), "intentTestingUser@email.com");
+        solo.enterText((EditText) solo.getView(R.id.loginEditTextPassword), "intentTesting");
+
+        solo.clickOnButton("Login");
+
+        // Asserts that the current activity is the RootActivity.
+        solo.assertCurrentActivity("Wrong Activity", RootActivity.class);
+
+        // Returns True if you can find "My Books" on the screen. Waits 50 seconds to find
+        // at least 1 match. This is to counter potentially long wait times when logging in.
+        assertTrue(solo.waitForText("My Books", 1, 50000));
+    }
+
+    /**
+     * Log in with the second intentTesting account.
+     * @param solo The solo object.
+     */
+    public static void login2(Solo solo) {
```

Comment:
login2 is the same as login1, can we pass the email and password as an argument?

Reply:
fixed